### PR TITLE
Add ability to submit login form and login. Currently redirect to len…

### PR DESCRIPTION
**Pull request by mcstover**
…d-vue?page=2

### CODE REVIEW

File: src/pages/LoginAndRegister/LoginAndRegister.vue [hunk 1]
```diff
@@ -23,12 +26,26 @@ export default {
...
+	padding: 1.625rem rem-calc(10);
+}
+@include breakpoint(medium) {
+	padding: 1.625rem 0;
+}
+
+// Control form layouts from parent to allow independence in other layouts
+.login-form,
+.register-form {
+	max-width: 280px;
```
Comment (BoulderBrains) [inline src/pages/LoginAndRegister/LoginAndRegister.vue]: Should probably make this max-width: rem-calc(280);

File: src/components/Forms/LoginForm.vue [hunk 2]
```diff
@@ -32,33 +35,133 @@
...
+					'Content-Type': 'application/x-www-form-urlencoded',
+				},
+				redirect: 'follow', // manual, *follow, error
+				referrer: 'no-referrer', // no-referrer, *client
+				// convert parameters into string ie. key=value&key=value...
+				body: parameters.join('&')
+			})
+				.then(response => {
+					// The response will be the doneUrl if passed or the current url page loaded
+					console.log(response);
```
Comment (BoulderBrains) [inline src/components/Forms/LoginForm.vue]: Do we want to leave this console.log in here?

Comment (mcstover) [reply src/components/Forms/LoginForm.vue]: no we prob don't need that lol

File: src/pages/LoginAndRegister/LoginAndRegister.vue (hunk 1 above)
Comment (mcstover) [reply src/pages/LoginAndRegister/LoginAndRegister.vue]: can do

**Pull request merged**